packages/yourloops/test/integration/mock/raw-data.mock.ts

```diff
@@ -0,0 +1,430 @@
+/*
```

Comment:
Folder could be:
`packages/yourloops/test/integration/data/data-api.data.ts`

What do you think of this?